### PR TITLE
Update tooltips to use as many BS vars as possible

### PR DESCRIPTION
**Pull request by DHedgecock**
## Description

Updates tooltips structure and theme styles to use as many bootstrap variables as possible. Makes extending Radical with Bootstrap a little easier.

## Tests

- [ ] I added tests for changes I made
- [x] All tests are _definitely_ passing

### CODE REVIEW

File: app/styles/ember-radical/component-structures/_tooltips.scss [hunk 1]
```diff
@@ -21,41 +21,33 @@
...
-  width: 500px;
+  width: $tooltip-max-width;
   position: absolute;
 }
 
 // Tooltip Content
 .tooltip-content {
-  margin-top: 10px;
-  padding: 0.5rem;
+  margin-top: $tooltip-arrow-width;
```
Comment (evrowe) [inline app/styles/ember-radical/component-structures/_tooltips.scss]: You've got THEME vars all up in your structural SCSS! That just ain't gonna work!

File: app/styles/ember-radical/skeleton-theme/utils/_variables.scss [hunk 2]
```diff
@@ -171,8 +172,15 @@ $modal-primary-bg: $brand-primary;
 
 // Tooltips
 // ---------------------------------------------------------------------------
-$tooltip-bg: #42403d;
+$tooltip-max-width: 200px;
 $tooltip-color: #fff;
+$tooltip-bg: #42403d;
```
Comment (evrowe) [inline app/styles/ember-radical/skeleton-theme/utils/_variables.scss]: Any of these variables that need to be used by the structural styles need to either go into the tooltip structural SCSS file, or a new _variables SCSS file needs to be made within the structural styles

Comment (DHedgecock) [inline app/styles/ember-radical/skeleton-theme/utils/_variables.scss]: GOOOOOOOOOOOOOOOD Catch

**Comment by DHedgecock** [comment]
Updated with a new structures vars file.

File: app/styles/ember-radical/skeleton-theme/utils/_variables.scss (hunk 2 above)
Comment (evrowe) [inline app/styles/ember-radical/skeleton-theme/utils/_variables.scss]: I see the old vars are still hanging out in the theme file… I assume then that you're intentionally overriding them for the theme?

**Pull request merged**